pkg/filter/filter.go

```diff
@@ -81,27 +82,28 @@ func (f *Filter) ShouldIgnoreTable(db, tbl string) bool {
 	if isSysSchema(db) {
 		return true
 	}
-	return !f.filter.MatchTable(db, tbl)
+	return !f.tableFilter.MatchTable(db, tbl)
 }
 
-// ShouldIgnoreDMLEvent removes DMLs that's not wanted by this change feed.
+// ShouldIgnoreDMLEvent removes DMLs that's not wanted by this changefeed.
 // CDC only supports filtering by database/table now.
 func (f *Filter) ShouldIgnoreDMLEvent(ts uint64, schema, table string) bool {
 	return f.shouldIgnoreStartTs(ts) || f.ShouldIgnoreTable(schema, table)
 }
 
-// ShouldIgnoreDDLEvent removes DDLs that's not wanted by this change feed.
+// ShouldIgnoreDDLEvent removes DDLs that's not wanted by this changefeed.
 // CDC only supports filtering by database/table now.
-func (f *Filter) ShouldIgnoreDDLEvent(ts uint64, ddlType model.ActionType, schema, table string) bool {
+func (f *Filter) ShouldIgnoreDDLEvent(ddl *model.Job) bool {
```

Comment:
For the DDL job like 'rename table'  which involves more than one table, the `SchemaName` and `TableName` of model.Job is not accurate to reflect the correct schema/table names of individual tables. So it's better to use `model.DDLEvent` as the input parameter.

Reply:
fixed.